### ISSUE TITLE
Evo 2.2 Audio Stream Won't Stop

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tripodtr/npr-android-app#36]
```
What steps will reproduce the problem?
1.Start the app
2.Go to a local station and press listen
3.Use menu/close.  The audio never stops.  The only way I've been able to stop 
the stream is to completely shut down the phone.

What is the expected output? What do you see instead?
The audio stream should stop when you select close.

What version of the product are you using? On what operating system?
1.3.1 on 2.2 (Evo)

Please provide any additional information below.

```

Original issue reported on code.google.com by `Ken98...@gmail.com` on 20 Aug 2010 at 11:50

**Comment by GoogleCodeExporter** [comment]
```
If you press Pause before using the menu close option, it will stop.  However, 
if you select menu/close then Pause becomes disabled and won't stop (no matter 
what you do)
```

Original comment by `Ken98...@gmail.com` on 20 Aug 2010 at 11:52

**Comment by GoogleCodeExporter** [comment]
```
Selecting close only closes the UI but the player is left running to play the 
content you selected.  If you open the application again, the player will show 
you what you are listening to and you will be able to stop/pause the stream.

The proposed UI for the next major revision does not have a close menu item.

```

Original comment by `marshgos...@gmail.com` on 23 Aug 2010 at 2:54

**Comment by GoogleCodeExporter** [comment]
```
I believe this will be addressed in a redesigned interaction via a 2.0 release 
in the near future, so I'm going to close it out.
```

Original comment by `jpenn...@gmail.com` on 23 Sep 2010 at 5:35
- Changed state: **WontFix**
